contracts/interfaces/HegicInterface.sol

```diff
@@ -88,6 +89,10 @@ interface IHegicBTCOptions is IHegicOptions {
         );
 }
 
-interface IHegicRewards {
-  function getReward(uint optionId) external;
+abstract contract IHegicRewards {
```

Comment:
abstract contract > interface for storing variables of other contracts

Reply:
This is fine for now. But just a note, you can actually do,

```
interface IHegicRewards {
  function rewardsRate() returns (uint256);
  function rewardedOptions(uint) external returns(bool);
}
```

Variables & mappings are just getters when calling external contracts.